testsuite/tests/kuadrant/authorino/operator/http/conftest.py

```diff
@@ -0,0 +1,48 @@
+"""Conftest for all tests requiring custom deployment of Authorino"""
+import pytest
+from weakget import weakget
+
+from testsuite.openshift.objects.authorino import AuthorinoCR
+from testsuite.objects import Authorino, Authorization
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.httpx import HttpxBackoffClient
+
+HTTP_DEFAULT_PORT=5001
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization(authorization, authorino, wildcard_domain, blame, openshift, label, rhsso_service_info) -> Authorization:
+    """In case of Authorino, AuthConfig used for authorization"""
+    #if authorization is None:
+    authorization = AuthConfig.create_instance(openshift, blame("ac"), wildcard_domain, labels={"testRun": label})
+    authorization.add_oidc_identity("rhsso", rhsso_service_info.issuer_url())
+    authorization.add_response(inject_json={'name': 'another-json-returned-in-a-header',
+                                'wrapperKey': 'x-ext-auth-other-json',
+                                'json': { 'properties': [{'name': 'propX', 'value': 'valueX'}]}
+                                })
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def client_http_auth(authorization, authorino_route):
+    """Returns httpx client to be used for requests, it also commits AuthConfig"""
+    authorization.commit()
+    client = HttpxBackoffClient(base_url=f"https://{authorino_route.as_dict()['spec']['host']}", verify=False)
+    yield client
+    client.close()
+    authorization.delete()
+
+
+@pytest.fixture(scope="module")
+def authorino_route(authorino, blame, openshift):
+    name = f"route-{authorino.name()}"
+    # expose cannot be used, see https://github.com/openshift/origin/issues/8699
+    route = openshift.do_action('create', 'route', 'edge', name,
```

Comment:
I do not understand why expose won't work, because that issue seems to be closed. I believe binding it to a specific service port should work

Reply:
Also, you could try using `openshift.routes.expose`

Reply:
It still doesn't work, see:
`$ oc expose service --name=route-authorin-mkudlej-mkdlej-kd-bzb23 -o json authorin-mkudlej-mkdlej-kd-bzb2-authorino-authorization --target-port=5001 --port=80

error: cannot use --target-port with --generator=route/v1`
I don't know what do you mean by `Also, you could try using openshift.routes.expose`, there is no such submodule of openshift client named `routes.expose` or maybe I haven't found it.